File: 06/HF/FizzBuzz.java
public class FizzBuzz {
    private int szam;

    public FizzBuzz(int szam) {
        this.szam = szam;
    }

    public void start() {
        for (int i = 1; i <= szam; i++) {
            StringBuffer sb = new StringBuffer();
            if (i % 3 == 0) {
                sb.append("fizz");
            }

            if (i % 5 == 0) {
                sb.append("buzz");
            }

            if (sb.toString().equals("")) {
                sb.append(Integer.toString(i));
            }

            System.out.println(sb.toString());
        }
    }
}

File: 13/HF/FizzBuzz/FizzBuzz.cs
using System;
using System.Text;

namespace FizzBuzz
{
    public class FizzBuzz
    {
        public int num;
        
        public FizzBuzz(int num)
        {
            this.num = num;
        }

        public void Start()
        {
            for (int i = 1; i <= num; i++)
            {
                StringBuilder sb = new StringBuilder();

                if (i % 3 == 0)
                {
                    sb.Append("fizz");
                }
                
                if (i % 5 == 0)
                { 
                    sb.Append("buzz");
                }

                if (sb.ToString().Equals(""))
                {
                    sb.Append(i);
                }
                
                Console.WriteLine(sb.ToString());
            }
        }
    }
}